CONTRIBUTING.md

```diff
@@ -24,14 +24,23 @@ it takes a bit longer. For more details, read below.
 
 ## Preparing the environment
 
-To reformat the code, check for common problems, and
-run the tests, it can be useful to prepare a
-[virtual environment](https://docs.python.org/3/tutorial/venv.html) and install
-certain libraries typeshed uses to check stub files.
+### Code away!
 
-Follow platform-specific instructions below. Following that, to automatically
-check your code before committing, you can copy the file `pre-commit` to
-`.git/hooks/pre-commit`.
+This repository has a Continuous Integration (CI) running on the pull requests,
+which includes automatically adjusting code formatting (`black`, `isort`) and
+running the tests. It means you can ignore all this on your side, focus on the
+code and rely on the CI to fix everything, or point you to the places that need
+fixing.
```

Comment:
```suggestion
Typeshed runs continuous integration (CI) on all pull requests. This will
automatically fix formatting (using `black`, `isort`) and run tests.
It means you can ignore all local setup on your side, focus on the
code and rely on the CI to fix everything, or point you to the places that
need fixing.
```